helpers/config.py

```diff
@@ -765,16 +775,50 @@ def __questions_aws(self):
             'Do you want to use AWS S3 storage?',
             default=self.__dict['use_aws']
         )
+
+        MAXIMUM_AWS_CREDENTIAL_ATTEMPTS = 2
+        aws_credential_attempts = 0
         if self.__dict['use_aws'] is True:
-            self.__dict['aws_access_key'] = CLI.colored_input(
-                'AWS Access Key', CLI.COLOR_QUESTION,
-                self.__dict['aws_access_key'])
-            self.__dict['aws_secret_key'] = CLI.colored_input(
-                'AWS Secret Key', CLI.COLOR_QUESTION,
-                self.__dict['aws_secret_key'])
-            self.__dict['aws_bucket_name'] = CLI.colored_input(
-                'AWS Bucket name', CLI.COLOR_QUESTION,
-                self.__dict['aws_bucket_name'])
+            while (
+                not self.__dict.get('aws_credentials_valid')
+            ) and (aws_credential_attempts <= MAXIMUM_AWS_CREDENTIAL_ATTEMPTS):
+                self.__dict['aws_access_key'] = CLI.colored_input(
+                    'AWS Access Key', CLI.COLOR_QUESTION,
+                    self.__dict['aws_access_key'])
+                self.__dict['aws_secret_key'] = CLI.colored_input(
+                    'AWS Secret Key', CLI.COLOR_QUESTION,
+                    self.__dict['aws_secret_key'])
+                self.__dict['aws_bucket_name'] = CLI.colored_input(
+                    'AWS Bucket name', CLI.COLOR_QUESTION,
+                    self.__dict['aws_bucket_name'])
+
+                aws_credential_attempts += 1
+                self.validate_aws_credentials()
+                attempts_remaining = (
+                    MAXIMUM_AWS_CREDENTIAL_ATTEMPTS - aws_credential_attempts
+                )
+                if (not self.__dict.get('aws_credentials_valid')) and (
+                    attempts_remaining >= 0
+                ):
+                    CLI.colored_print(
+                        f'Attempts remaining: {attempts_remaining}',
```

Comment:
Maybe a more verbose message would be more explicit?

Reply:
💯 I also wasn't sure if this was totally necessary, but maybe a nice UX feature for feedback to the user.